drivers/dma/dma_dw_common.c

```diff
@@ -89,7 +89,7 @@ void dw_dma_isr(const struct device *dev)
 			LOG_DBG("Dispatching transfer callback");
 			chan_data->dma_tfrcallback(dev,
 						   chan_data->tfruser_data,
-						   channel, 0);
+						   channel, DMA_STATUS_FULL);
```

Comment:
The define here doesn't really match up with the status, this is a transfer complete and STATUS_FULL in terminology feels different